<launch>
   <node pkg="rosbag" type="play" name="rosbag" required="true" args="/media/joe/My\ Passport/data/1camtest.bag"/>
   <node name="extract" pkg="image_view" type="extract_images" respawn="false" required="true" output="screen" cwd="ROS_HOME">
     <remap from="image" to="/cameras/cam1"/>
   </node>
</launch>
